src/runtime_src/core/pcie/tools/cloud-daemon/pciefunc.h

```diff
@@ -21,7 +21,7 @@
 
 #include <string>
 #include <mutex>
-#include "core/pcie/linux/scan.h"
+#include "core/pcie/linux/pcidev.h"
```

Comment:
Sort.

Reply:
By header path length? By header filename?

Reply:
By lexicographic alphabetic order.
Header path length would be ambiguous. :-)

Reply:
Sure. And also follow Soren's grouping rule.